.travis.yml

```diff
@@ -3,20 +3,33 @@ language: "php"
 php:
     - "7.1"
     - "7.2"
+    - "7.3"
+
+matrix:
+    include:
+        - php: '7.1'
+          env: PREFER_LOWEST=1
+        - php: '7.2'
+          env: PREFER_LOWEST=1
+        - php: '7.3'
+          env: PREFER_LOWEST=1
 
 before_install:
     - "composer self-update"
 
 install:
-    - "composer install"
+    - if [[ $PREFER_LOWEST != 1 ]]; then composer update --prefer-dist --prefer-stable --no-interaction; fi
```

Comment:
Is it really necessary to run `composer update` when `$PREFER_LOWEST != 1`? Please use `composer install --prefer-dist --no-interaction` instead.

Reply:
I tried that, see commit 977aea7 (using composer install and failing) and 7aa518b (using composer update and passing).

I'm not sure if it's valuable to commit the lock anyway for a library (see https://getcomposer.org/doc/02-libraries.md#lock-file). In the end i think it makes sense to test against the lowest and highest possible versions and not against the version someone once updated or installed.

Reply:
Okay. So then please remove the lock file and put it under `.gitignore`.

Reply:
Thank you! I didn't do this to keep the scope as small as possible but i like it much better without the `composer.lock` file.